### ISSUE TITLE
Select Responsive Design Framework

### ISSUE THREAD

**Issue by KatochAarushi** [nmohamm704/UNCC-Food-Recommendation#7]
Chosen framework does not meet the design requirements for responsiveness.

Framework introduces issues with cross-browser compatibility.

**Comment by nmohamm704** [comment]
Backend Design framework complete
